### PR TITLE
Run in port 80 and minor changes in package

### PR DESCRIPTION
**Pull request by Rub21**
@aaronlidman  I've done minor changes  in to-fix-backend  to run this in a ECS.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "aws-sdk": "^2.2.43",
     "boom": "^2.6.1",
     "csv-parser": "^1.5.0",
+    "decrypt-kms-env": "^2.0.1",
```
Comment (aaronlidman) [inline package.json]: @Rub21 - Is this needed here?

Comment (Rub21) [reply package.json]: @aaronlidman  I am using it to pass the envs to the app when it is runing in docker. 
`CMD eval $(./to-fix-backend/node_modules/.bin/decrypt-kms-env) && cd to-fix-backend/ && npm start`. I am  looking to move this   in the build repository.

Comment (aaronlidman) [reply package.json]: 👍  Sounds better in the build repo.

Comment (Rub21) [reply package.json]: I've added `decrypt-kms-env` in build repo.

**Pull request merged**